.ci_support/linux_64_openssl1.1.1.yaml

```diff
@@ -1,3 +1,8 @@
+arrow_cpp:
+- 6.0.2
+- 7.0.1
+- 8.0.1
+- 9.0.0
```

Comment:
> I looked at a random build and found this:
>
> ```
> "libgdal-arrow-parquet-3.6.0-h0f717e7_": {
>   "recipe": {
>     "arrow_cpp": "6.0.2"
> ```

That's because by default you're getting builds for 4 different arrow versions. You can add a skip to the libgdal-arrow-parquet output so that it only builds for arrow 10.x (and then wait for the [migrator](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3793) you already referenced)

Reply:
I tried to add one, but conda smithy rerender wasn't happy. Probably because I'm a conda-forge noob and didn't get things right. I just went to adding libarrow only without version constraints